src/first.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/**
+ * @module utils/first
+ */
+
+/**
+ * Returns first item of an `iterable`.
+ *
+ * @param {Iterable.<*>} iterable
+ * @returns {*}
+ */
+export default function first( iterable ) {
+	const iteratorItem = iterable.next();
+
+	if ( !iteratorItem.done ) {
```

Comment:
I mean, the code without `!` is usually a little bit simpler to read than with it.

Second, usual construction is:
```
    if (!somethingIsWrong)
        return;

    // Do something default
```